R/tag_query.R

```diff
@@ -713,144 +580,142 @@ tagQuery_ <- function(
-        #' method before calling `$append(...)` to replace all selected
-        #' elements' children.
+        #' * `$empty()`: Remove any children of each selected tag. Use this
+        #' method before calling `$append(...)` to replace the children of
+        #' each selected tag, with other content.
         empty = function() {
           rebuild_()
           tagQueryChildrenEmpty(selected_)
           # no need to rebuild_
-          self
+          invisible(self)
         },
-        ## end Adjust Children
 
-        #' ## Adjust Siblings
+
+        #' ## Modify siblings
         #'
         #' * `$after(...)`: Add all `...` objects as siblings after each of the
-        #' selected elements.
+        #' selected tags.
         after = function(...) {
           rebuild_()
           tagQuerySiblingAfter(selected_, ...)
           rebuild_()
-          self
+          invisible(self)
         },
         #' * `$before(...)`: Add all `...` objects as siblings before each of
-        #' the selected elements.
+        #' the selected tags.
         before = function(...) {
           rebuild_()
           tagQuerySiblingBefore(selected_, ...)
           rebuild_()
-          self
+          invisible(self)
         },
-        #' * `$replaceWith(...)`: Replace all selected elements with `...`. This
-        #' also sets the selected elements to an empty set. A new `tagQuery()`
-        #' object will be created with an empty set of selected elements.
+        #' * `$replaceWith(...)`: Replace all selected tags with `...` in the
+        #' root tag and clear the selection.
         replaceWith = function(...) {
           rebuild_()
           tagQuerySiblingReplaceWith(selected_, ...)
           newTagQuery(list(), rebuild = TRUE)
         },
-        #' * `$remove(...)`: Remove all selected elements from the `tagQuery()`
-        #' object. The selected elements is set to an empty set. A new
-        #' `tagQuery()` object will be created with an empty set of selected
-        #' elements.
+        #' * `$remove(...)`: Remove all selected tags from the root tag and
+        #' clear the current selection.
         remove = function() {
           rebuild_()
           tagQuerySiblingRemove(selected_)
           # Remove items from selected info
           newTagQuery(list(), rebuild = TRUE)
         },
-        ## end Adjust Siblings
 
-        ## Generic Methods
+
         #' ## Generic methods
         #'
-        #' * `$each(fn)`: Perform function `fn` on each of the selected
-        #' elements. `fn` should accept two arguments: a selected element and
+        #' * `$each(fn)`: Modify each selected tag with a function `fn`.
+        #' `fn` should accept two arguments: a selected element and
         #' the selected element's position within the selected elements. This
-        #' argument order is different than jQuery's `$().each()` as there is no
+        #' argument order mimics jQuery's `$().each()` as there is no
```

Comment:
I'll clean this up once I actually understand how `$each()` works. Is this expected?

```r
> (tagQuery(div(div(), div()))$find("div")$each(function(x, i) x <- a()))
Root:
<div>
  <div></div>
  <div></div>
</div>

Selected:
[[1]]
<div></div>
[[2]]
<div></div>
```